app/assets/javascripts/latlong_map.coffee

```diff
@@ -1,70 +0,0 @@
-marker = null
-
-# fallback to PVTA if we can't geocode the incident's location
-PVTA = { lat: 42.105552, lng: -72.596511 }
-
-createMap = (mapSelector, latLng, newValues) ->
-  map = new google.maps.Map mapSelector[0], { zoom: 15, tilt: 0, center: latLng }
-  placeMarker map, latLng, newValues
-  google.maps.event.addListener map, 'click', (event) ->
-    latLong = { lat: event.latLng.lat(), lng: event.latLng.lng() }
-    placeMarker map, latLong, true
-
-
-fillLatLngFields = (latLng) ->
-  $('#incident_latitude').val Number(latLng.lat.toFixed(5))
-  $('#incident_longitude').val Number(latLng.lng.toFixed(5))
-
-initLatLngMap = (mapSelector) ->
-  if mapSelector[0].dataset.lat
-    lat = parseFloat mapSelector.data('lat')
-    lng = parseFloat mapSelector.data('lng')
-    createMap mapSelector, { lat: lat, lng: lng }, false
-  else
-    location = mapSelector.data 'location'
-    geocoder = new google.maps.Geocoder()
-    geocoder.geocode address: location, (results, status) ->
-      if status == google.maps.GeocoderStatus.OK
-        result = results[0]
-        createMap mapSelector, result.geometry.location, false
-      else
-        $('.no-geocode-alert').slideDown()
-        createMap mapSelector, PVTA, false
```

Comment:
Oh, I didn't end up preserving this feature, which is basically "If we don't have an initial lat/long, try to geocode what's already in the address fields and use that"

It didn't seem like a needed feature based on what I understand about the workflows, so I didn't bother with it